docs/resources/secondary_connection.md

```diff
@@ -28,9 +28,9 @@ resource "snowflake_secondary_connection" "complete" {
 }
 ```
 
--> **Note** Instead of using fully_qualified_name, you can reference objects managed outside Terraform by constructing a correct ID, consult [identifiers guide](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/guides/identifiers#new-computed-fully-qualified-name-field-in-resources).
+-> **Note** Instead of using fully_qualified_name, you can reference objects managed outside Terraform by constructing a correct ID, consult [identifiers guide](../guides/identifiers#new-computed-fully-qualified-name-field-in-resources).
```

Comment:
not sure if this one will be correct

Reply:
It's [404](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/resources/docs/guides/identifiers#new-computed-fully-qualified-name-field-in-resources).

Reply:
it works for me on terraform registry but does not work on github